### PR TITLE
Add Max_Fee as the optional for gas limit.

### PR DESCRIPTION
**Pull request by superboyiii**
Add Max_Fee  as the optional for gas limit.

### CODE REVIEW

File: RpcWallet/RpcWallet.cs [hunk 1]
```diff
@@ -307,8 +308,25 @@ private JObject SendFrom(UIntBase assetId, UInt160 from, UInt160 to, string valu
                     ScriptHash = to
                 }
             }, from: from, change_address: change_address, fee: fee);
+            if (tx.Size > 1024)
+            {
+                fee += Fixed8.FromDecimal(tx.Size * 0.00001m + 0.001m);
```
Comment (erikzhang) [inline RpcWallet/RpcWallet.cs]: Maybe `fee = max(fee, Fixed8.FromDecimal(tx.Size * 0.00001m + 0.001m))`?

Comment (superboyiii) [reply RpcWallet/RpcWallet.cs]: Fine.

**Comment by erikzhang** [comment]
Please resolve the conflicts.

**Comment by superboyiii** [comment]
Done.

**Pull request merged**